src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -382,7 +382,7 @@ extension HomeInteractor {
 extension HomeInteractor {
 	func updateTestResults() {
 		// Avoid unnecessary loading.
-		guard testResult == nil || testResult == .pending else { return }
+		guard testResult == nil || testResult != .positive else { return }
```

Comment:
Once the state .positive is reached, the app goes to end-of-life state (and should _not_ update the result further). Seems to be correct now!